src/components/organisms/AdminVenueView/components/AdminCardTitle/AdminCardTitle.scss

```diff
@@ -0,0 +1,16 @@
+@import "scss/constants";
+
+$height: 1rem;
```

Comment:
[NOTE]

Using functions like `font-size--parent(1)` or `font-size--root(1)` adds more semantics (makes it more obvious what the intent was.

Reply:
it's kinda confusing that a function which name starts with `font-size` is being applied to css attributes that relate to height (eg. `min-height`, `line-height`) prob we can figure out another name? not sure